tests/unit/policyEvaluator.js

```diff
@@ -58,9 +53,9 @@ describe('policyEvaluator', () => {
             it('should grant access under multi-statement ' +
                 'policy if action and resource match those in policy',
                 () => {
-                    requestContext.action = 's3:GetObject';
-                    requestContext.resource =
-                        'arn:aws:s3:::notVeryPrivate/object';
+                    requestContext._apiMethod = 'objectGet';
```

Comment:
I would recommended a function for this as well. It would be easier to change any method names or add new ones in one place in the future.